assignments/prototypes.js

```diff
@@ -14,15 +14,26 @@
   * dimensions
   * destroy() // prototype method -> returns the string 'Object was removed from the game.'
 */
-
+function GameObject(param) {
+  this.createdAt=param.createdAt;
+  this.dimensions=param.dimensions;
+}
+GameObject.prototype.destroy=function(){return 'Object was removed from the game';};
 /*
   === CharacterStats ===
   * hp
   * name
   * takeDamage() // prototype method -> returns the string '<object name> took damage.'
   * should inherit destroy() from GameObject's prototype
 */
-
+function CharacterStats(param) {
+  GameObject.call(this,param)
+  this.hp=param.hp;
+  this.name=param.name;
+}
+CharacterStats.prototype=Object.create(GameObject.prototype);
+CharacterStats.prototype.takeDamage=function(){return `${this.name} took damage.`;}
+CharacterStats.prototype.destroy=function() {return `${this.name} was removed from the game`;};
```

Comment:
Thanks for mentioning this part. Great job on the tasklist and frequent commits! 👍